cmd/run.go

```diff
@@ -274,11 +281,34 @@ func pollTask(c *client.Client, id string, timeout time.Duration, showlogs, remo
 
 		fmt.Println("ID: ", item.ID, " Update at: ", item.UpdatedAt)
 		for {
+			if len(taskRequest.HeartbeatUrl) > 0 && len(taskRequest.JobToken) > 0 {
+				heartbeatClient := &http.Client{}
+				heartbeatRequest, hrError := http.NewRequest(http.MethodPut, taskRequest.HeartbeatUrl, bytes.NewBufferString("{}"))
+				if hrError != nil {
+					log.Fatal(hrError)
```

Comment:
`http.NewRequest` returns an error if method or  URL is not valid. 
Do we want to exit in this case or should we continue the job and just log? That's a decision we need to take here.